pilot/pkg/bootstrap/sidecarinjector.go

```diff
@@ -99,7 +99,7 @@ func (s *Server) initSidecarInjector(args *PilotArgs) (*inject.Webhook, error) {
 				return nil
 			}
 
-			patcher.Run(stop)
+			go patcher.Run(stop)
```

Comment:
unrelated?

Reply:
Just keeping consistent pattern for controllers to have Run block